### PR TITLE
Features/group accounts

### PR DESCRIPTION
**Pull request by WesVW**
Testing procedure:

Log in with 
wesley_van_wyk@hotmail.com
Test123!*1

Click on Network and load the contacts page. Contacts should appear and contact requests should be stated.

### CODE REVIEW

File: src/Accounts/Views/Network/Contacts.cshtml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+<div class="columns is-variable is-5">
+    <div class="column is-3">
+        <div class="columns">
+            <div class="column is-12">
+                <div class="card has-background-white">
+                    <div class="card-content">
+                        <table class="table is-hoverable">
+                            <thead>
+                                <tr>
+                                    <th colspan="2"><h5>My Network</h5></th>
```
Comment (vertechcon) [inline src/Accounts/Views/Network/Contacts.cshtml:15]: My network seems to be intended to work the same way across all three views, it should probably be a single partial view

File: src/Accounts/Views/Network/Groups.cshtml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+﻿@model CommunAxiom.Accounts.ViewModels.Network.ManageViewModel
+@{
+    ViewData["Title"] = "Manage Network";
+}
+<div class="hero is-halfheight">
```
Comment (vertechcon) [inline src/Accounts/Views/Network/Groups.cshtml:5]: Page is not responsive. 
The design should also be similar across all three Network pages. Contacts uses Cards. 
Hero is typically used for media content, not for data driven content.
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_hero_image

File: src/Accounts/Views/Network/Requests.cshtml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+﻿@model CommunAxiom.Accounts.ViewModels.Network.ManageViewModel
+@{
+    ViewData["Title"] = "Manage Network";
+}
+<div class="hero is-halfheight">
```
Comment (vertechcon) [inline src/Accounts/Views/Network/Requests.cshtml:5]: Same as for Groups

**Review by vertechcon — CHANGES_REQUESTED** [review]
Couple of comments in the code, most of the corrections are in the views.

File: src/Accounts/Accounts.csproj [hunk 4]
```diff
@@ -6,6 +6,13 @@
     <RootNamespace>CommunAxiom.Accounts</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
```
[the remark this answers is not in the document]
Comment (WesVW) [reply src/Accounts/Accounts.csproj:9]: this item group pointed to folders i created that should have been deleted. I have removed this.

File: src/Accounts/Views/Network/Contacts.cshtml (hunk 1 above)
Comment (WesVW) [reply src/Accounts/Views/Network/Contacts.cshtml:15]: created a partial view with the card and the table. implemented in all three views

File: src/Accounts/Views/Network/Groups.cshtml (hunk 2 above)
Comment (WesVW) [reply src/Accounts/Views/Network/Groups.cshtml:5]: implemented groups in the same way as contacts was created. page is now responsive. Initially this checkin was to publish contacts and i was going to address groups later.

I have made the necessary adjustments and implemented groups

File: src/Accounts/Views/Network/Requests.cshtml (hunk 3 above)
Comment (WesVW) [reply src/Accounts/Views/Network/Requests.cshtml:5]: implemented the same as contacts

File: src/Accounts/Views/Network/Manage.cshtml [hunk 5]
```diff
@@ -0,0 +1,83 @@
+﻿@model CommunAxiom.Accounts.ViewModels.Network.ManageViewModel
+@{
+    ViewData["Title"] = "Manage Network";
+}
+<div class="hero is-halfheight">
```
[the remark this answers is not in the document]
Comment (WesVW) [reply src/Accounts/Views/Network/Manage.cshtml:5]: page deleted

**Review by vertechcon — APPROVED** [review]

**Pull request merged**